AI/app/rag/attraction_search.py

```
@@ -13,22 +13,21 @@
+    vectordb = FAISS.load_local(
+        folder_path=FAISS_DB_PATH,
+        embeddings=embedding_function,
+        allow_dangerous_deserialization=True,  # 주의: 신뢰할 수 있는 소스에서만 True로 설정
     )
 
+    retriever = vectordb.as_retriever(
+        search_type="similarity",
+        search_kwargs={"k": 30, "lambda_mult": 0.6, "score_threshold": 0.8},
+    )
+except Exception as e:
+    raise RuntimeError(f"FAISS 벡터 데이터베이스 로드 실패: {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)